libvmi/driver/xen/xen_events_private.h

```diff
@@ -144,6 +144,15 @@ status_t xen_set_cpuid_event(vmi_instance_t vmi, bool enabled);
 status_t xen_start_single_step(vmi_instance_t vmi, single_step_event_t *event);
 status_t xen_stop_single_step(vmi_instance_t vmi, uint32_t vcpu);
 status_t xen_shutdown_single_step(vmi_instance_t vmi);
+status_t xen_altp2m_get_domain_state (vmi_instance_t vmi, bool *state);
```

Comment:
Should be moved to another header, altp2m_private.h which gets included in xen.c